server/src/main/java/io/pravega/schemaregistry/storage/impl/schemas/PravegaKeyValueSchemas.java

```diff
@@ -132,6 +134,7 @@ public PravegaKeyValueSchemas(TableStore tableStore) {
             entries.put(KEY_SERIALIZER.toBytes(new SchemaIdChunkKey(id, i)),
                     new VersionedRecord<>(bytes, null));
         }
+        log.info("Calling update entries for new schema addition");
```

Comment:
can we add info/identifier about the schemaInfo being added?

Reply:
I'd recommend removing this log or changing it to trace level

Reply:
Changed to trace with schemaInfo